app/controllers/api/plans_base_controller.rb

```diff
@@ -28,8 +28,9 @@ def authorize_action
     authorize! :create, :plans
   end
 
-  def resource(id = params[:id])
-    collection.readonly(false).find_by_id(id)
+  def resource(id = params[:id].presence)
```

Comment:
wouldn't it raise with the `.presence` ?

Reply:
@guicassolato

Reply:
No basically `#presence` works with `nil`

It does something like:

```ruby
def presence
  present? ? self : nil
end

### e.g

"".presence # => nil
nil.presence # => nil
1.presence # => 1
Object.new.presence # => #<Object:0x00007fcc731ffd78>
```